app/models/post.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  validates :links, presence: true
+  validates :user_id, presence: true
```

Comment:
- [ ] Same here for `user_id`.
**TIP** If your validations share the same validator i.e. (presence), you can combine them in the same line.
`validates :links, :title, presence: true`